Contributing.MD

```diff
@@ -0,0 +1,30 @@
+Contributing to README Template
+
+We welcome your contributions to README Template. This doc describes the process to contribute patches to README Template and the general guidelines we expect contributors to follow.
+Before You Start
+
+We accept patches in the form of github pull requests. If you are new to github, please read How to create github pull requests first.
+Contributor License Agreements
+
+Contributions to this project must be accompanied by a Contributor License Agreement. You (or your employer) retain the copyright to your contribution, this simply gives us permission to use and redistribute your contributions as part of the project.
```

Comment:
Need to check on this

Reply:
I do not think we should require CLAs. They are not recommended by the open source community, as they require burden on the contributor, and the norm in the open-source community is the understanding that contributions will be incorporated into the master code under the same license as the master code (also called inbound=outbound).

GitHub even explictly refers to inbound=outbound in their Terms: https://help.github.com/en/github/site-policy/github-terms-of-service#6-contributions-under-repository-license 

Basically, it is common knowledge by contributing to a repo that your contribution will be incorporated into the code under the existing license. No need to have contributors jump through extra hoops to determine this.

Recommend we incorporate language to the effect of "Contributions will be release in accordance with GitHub site policies (link to https://help.github.com/en/github/site-policy/github-terms-of-service#6-contributions-under-repository-license), including that contributions will be released under the same license applied to this repository.

Reply:
Way to do your homework and come up with creative solutions though - gives us more to think about for sure!